packages/@okta/vuepress-site/docs/reference/api/asa/attributes/index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Introduction to Attribute Endpoints
+category: asa
+---
+
+# Attributes API
+
+## Getting Started
+
+This article provides an overview of the attribute API
```

Comment:
The name of the API should be capitalized and consistent. For example, either "Attribute API" or "Attributes API".